src/main/java/org/example/mappers/JobRoleMapper.java

```diff
@@ -0,0 +1,26 @@
+package org.example.mappers;
+
+import org.example.models.OpenJobRoleResponse;
+import java.util.stream.Collectors;
+import java.util.List;
+
+public class JobRoleMapper {
+   private JobRoleMapper jobRoleMapper;
+    /*
+    maps the JobRoleResponse to a stream
+    and then uses Collectors to convert the stream to a List
+     */
+    public static List<OpenJobRoleResponse> mapJobRoleResponseList(
+            final List<OpenJobRoleResponse> jobRoleResponseList) {
```

Comment:
I don't think this does anything as you're mapping to the same type, you can just remove this class

Reply:
removed